crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -264,8 +264,13 @@ impl DiskFileSystem {
     }
 
     pub async fn to_sys_path(&self, fs_path: FileSystemPathVc) -> Result<PathBuf> {
-        let path = Path::new(&self.root).join(&*unix_to_sys(&fs_path.await?.path));
```

Comment:
Joining an empty path leads to a tailing `/` which is not matching the path reported from watcher